CHANGELOG.md

```diff
@@ -9,6 +9,35 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.3.0 - 2019-01-23
+
+### Added
+
+- Add documentation for OVS pipeline.
+- Add Network Policy information to monitoring CRDs for both the Agent and the Controller.
+- Add support for the IPsec ESP protocol for GRE tunnels only; it can be enabled by applying antrea-ipsec.yml instead of antrea.yml.
+- Add framework to develop CLI commands for Antrea; the antctl binary only supports the "version" command at the moment.
+- Add OpenFlow and OVSDB connection health information to the Agent's monitoring CRD.
```

Comment:
Would you put it after policy information in CRD?

Reply:
Done